src/core/Controller.cpp

```diff
@@ -117,11 +117,11 @@ float Controller::value( int offset )
 	{
 		updateValueBuffer();
 	}
-	return m_valueBuffer.values()[ offset ];
+	return m_valueBuffer.data()[offset];
```

Comment:
Since I believe `m_valueBuffer` in this context is just a regular `std::vector<float>`, you can use `operator[]` directly here rather than calling `data`.

```suggestion
	return m_valueBuffer[offset];
```